gluten-substrait/src/main/java/org/apache/gluten/substrait/rel/RelBuilder.java

```diff
@@ -148,8 +148,8 @@ public static RelNode makeAggregateRel(
   public static RelNode makeReadRel(
       List<TypeNode> types,
       List<String> names,
-      List<ColumnTypeNode> columnTypeNodes,
       ExpressionNode filter,
+      List<ColumnTypeNode> columnTypeNodes,
```

Comment:
Why you move the argument?

Reply:
Make the order of construct object consistent with parameters. This can help improve readability.